AWS/legos/aws_create_redshift_query/aws_create_redshift_query.jsonx

```diff
@@ -0,0 +1,38 @@
+{
```

Comment:
I think the jsonx file is not needed here, it is generated later...

Reply:
i didnt generate it - it was generated when I exported from Docker.